azure-pipelines.yml

```diff
@@ -0,0 +1,54 @@
+pr: none
+
+trigger:
+  branches:
+    include: ['*']
+  tags:
+    include: ['v*.*.*']
+
+variables:
+  - group: global-variables
+
+parameters:
+- name: 'awsRegion'
+  type: string
+  default: 'ap-southeast-2'
+
+- name: 'imageName'
+  type: string
+  default: 'public.ecr.aws/mx51io/eks-rolling-update-fork'
+
+
+stages:
+
+# Build and Release stage
+- stage: build_and_release
+  jobs:
+  - job: Build
+    pool: shared
+    steps:
+
+    - script: make setup && . venv/bin/activate
+      displayName: "Create venv and install packages"
+
+    - script: |
+        VERSION=$( ./determine-version.sh )
+        echo make docker-dist version=$VERSION
+        make docker-dist version=$VERSION
+      displayName: "Build docker image"
+
```

Comment:
Could you please add Twistlock scan if that's possible, thanks!

Reply:
Will do.